src/index.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
+
+'use strict'
+
+require('dotenv').config()
+const puppeteer = require('puppeteer');
```

Comment:
Unused import

Reply:
I should add linter and rules so those errors are shown before commit